features/polyExplorer/src/components/dataRegionsLegend/dataRegionsLegend.jsx

```diff
@@ -6,29 +6,31 @@ import i18n from "../../i18n.js";
 const DataRegionsLegend = () => {
     return (
         <div className="location-legend">
-            <div className="legend">
+            <div className="leg">
```

Comment:
Have you considered using some name that's more informative? `legend-wrapper` or `legends-and-stuff`. I don't really know, but "leg" is, well, a :leg:

Reply:
`legend-container` maybe?